src/utils/localStorageAPI.ts

```diff
@@ -22,8 +22,12 @@ export const LSAPIVerifyUser = (login: string, password: string): boolean => {
   );
 };
 
-export const LSAPISetFavouriteGenres = (newGenres: number[]): void => {
-  localStorage.setItem("favouriteGenres", JSON.stringify(newGenres));
+export const LSAPIToggleFavouriteGenre = (genreId: number): void => {
```

Comment:
IMO, it looks messy.

I would encapsulate inside `api/actions`.
```
api/actions/toggle-favourite-genre.ts
api/actions/toggle-favourite-movie.ts
api/actions/get-favourite-genres.ts
api/actions/get-genres.ts
```

```
// api/actions/toggle-favourite-genre.ts

export default async function toggleFavourite(genreId): Promise<void> {}
```

Then once having API to store such things I replace implementations of these actions and leave everything else as-is. Because other parts of the application don't dive into of the implementation.